client/src/main/scala/drt/client/components/AccessibilityStatementComponent.scala

```diff
@@ -0,0 +1,35 @@
+package drt.client.components
+
+import japgolly.scalajs.react.{Children, JsFnComponent}
+import japgolly.scalajs.react.vdom.VdomElement
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+@js.native
+trait IAccessibilityStatementProps extends js.Object {
+  var teamEmail: String = js.native
+  var emailUsToReportAProblem: js.Function0[Unit] = js.native
+}
+
+object IAccessibilityStatementProps {
+  def apply(teamEmail: String, emailUsToReportAProblemHandler: js.Function0[Unit]): IAccessibilityStatementProps = {
+    val p = (new js.Object).asInstanceOf[IAccessibilityStatementProps]
+    p.teamEmail = teamEmail
+    p.emailUsToReportAProblem = emailUsToReportAProblemHandler
+    p
+  }
+}
+
+object AccessibilityStatementComponent {
```

Comment:
We probably don't need to use the accessibility statement component in ScalaJs. If it's always in the `BottomBar` then it can be included in the BottomBar react component rather than here which will remove the need for the facade

Reply:
As discussed not needed